SQL/MySQL.sublime-syntax

```diff
@@ -662,6 +676,11 @@ contexts:
     - include: expressions
     - include: pop-on-top-level-reserved-word
 
+  variable-member:
+    - meta_content_scope: variable.other.member.mysql
+    - match: ''
+      pop: 1
```

Comment:
Exclude prototype and re-use `immediately-pop`?

```suggestion
    - meta_include_prototype: false
    - meta_content_scope: variable.other.member.mysql
    - include: immediately-pop          
```

Reply:
Maybe without the trailing spaces? :wink: